lib/aggregate/base.rb

```diff
@@ -28,6 +28,16 @@ def <=>(other)
       end || 0
     end
 
+    def self.secret_keys_from_config
+      if Aggregate.configuration.keys_list.is_a?(Array)
+        Aggregate.configuration.keys_list.map { |key| Base64.strict_decode64(key) }
+      elsif Aggregate.configuration.keys_list.is_a?(String)
+        [ Base64.strict_decode64(Aggregate.configuration.keys_list) ]
+      else
+        []
+      end
+    end
```

Comment:
This one liner should accomplish the same thing without type checking. Array(<obj>) will return either the same array, a new array with <obj>, or an empty array if nil is passed.
```ruby
Array(Aggregate.configuration.keys_list).map { |key| Base64.strict_decode64(key) }
```